nemo/collections/nlp/data/language_modeling/megatron/gpt_dataset.py

```diff
@@ -261,12 +263,12 @@ def build_dataset(index, name):
     return (train_dataset, valid_dataset, test_dataset)
 
 
-def get_indexed_dataset_(data_prefix, data_impl, skip_warmup):
+def get_indexed_dataset_(data_prefix, data_impl, skip_warmup, delay_data_mmap):
     """Build indexed dataset."""
     logging.info(' > building dataset index ...')
 
     start_time = time.time()
-    indexed_dataset = make_indexed_dataset(data_prefix, data_impl, skip_warmup)
+    indexed_dataset = make_indexed_dataset(data_prefix, data_impl, skip_warmup, delay_data_mmap=delay_data_mmap)
     logging.info(' > finished creating indexed dataset in {:4f} ' 'seconds'.format(time.time() - start_time))
     logging.info('    number of documents: {}'.format(indexed_dataset.sizes.shape[0]))
```

Comment:
These log statements only make sense if delay_data_mmap=False?

Reply:
The document, shuffle, and sample index files will be created even with `delay_data_mmap=False`. Only the `.bin` files of the dataset will not be accessed when `delay_data_mmap=True`